README.md

```diff
@@ -26,6 +26,23 @@ To start serving the site locally in development:
 make run
 ```
 
+To run locally in conjunction with [`identity-style-guide`](https://github.com/18F/identity-style-guide/), run the following commands:
+
+0. In the `identity-site` directory, run: `npm run link identity-style-guide`
+0. In the `identity-style-guide` directory, run `npm link`. This will create a symlink that will make changes to this repo accessible in `identity-site`
+
+Then, open two new terminal windows. In one window run the following to watch for site-specific JS changes:
```

Comment:
So in my mind, we should definitely not need two windows, I think our tooling should handle that for the most part.

The `Makefile.server` gets run with the `-j4` option which means it runs 4 dependencies in parallel, so I'd love it we could update our Makefiles so they do the parallelization? They should even be able to do the `npm run link` bit

Let me know if you want any help trying to set this up